### PR TITLE
feat: support to expose admin api

### PR DESCRIPTION
**Pull request by tokers**

### CODE REVIEW

File: chart/apisix/templates/configmap.yaml [hunk 1]
```diff
@@ -82,7 +82,9 @@ data:
         - 0.0.0.0/0
       {{- end}}
       #   - "::/64"
-      # port_admin: 9180              # use a separate port
+      {{- if .Values.admin.enabled }}
+      port_admin: {{ .Values.admin.containerPort }}
```
Comment (totemofwolf) [inline chart/apisix/templates/configmap.yaml:86]: Why not use `admin.port`

Comment (lilien1010) [reply chart/apisix/templates/configmap.yaml:86]: I used to call it admin_port

**Review by lilien1010 — APPROVED** [review]

File: chart/apisix/templates/configmap.yaml (hunk 1 above)
Comment (tokers) [reply chart/apisix/templates/configmap.yaml:86]: I think `.Values.admin.admin_port` is redundant, maybe `.Values.admin.port` is better.

Comment (tokers) [reply chart/apisix/templates/configmap.yaml]: @totemofwolf @lilien1010 Changed.

**Review by membphis — APPROVED** [review]

File: chart/apisix/values.yaml [hunk 2]
```diff
@@ -97,6 +97,15 @@ dns:
 admin:
   # Enable Admin API
   enabled: true
+  # admin service type
+  type: ClusterIP
+  # loadBalancerIP: a.b.c.d
+  # loadBalancerSourceRanges:
+  #   - "143.231.0.0/16"
```
Comment (shreemaan-abhishek) [inline chart/apisix/values.yaml:104]: @membphis why are these comments needed? 🤔

**Pull request merged**